contributing/admins.md

```diff
@@ -0,0 +1,38 @@
+
+- The mental wellbeing of our staff
+- The mental wellbeing of our maintainers and mentors
+- The needs and best interests of our students
+- The experience of our contributors and community
+
+We also place a high degree of faith in pragmatism - not always seeking the most "correct" or "perfect" outcome, but making decisions that make positive steps in the right directions, even if sometimes they come with some downsides.
+
+The role of our administrator (or "admins") is to ensure that in discussions and interactions around Exercism that these aims are met and that decisions are made pragmatically.
+
+From time to time an admin may come into a conversation and announce they are acting in that role, for example starting a post with "Wearing my admin hat...".
+They will normally link back to this post to explain what that means.
+While wearing that hat, they will often bring resolutions to conversations that do not seem to be coming to a natural consensus.
+For example, determining whether a code sample should be written using one idiom or another, or deciding whether to follow one style or another.
+
+Then generally try and resolve discussions by adhering to the following guidance:
+
+- Is there a clearly correct outcome that one side is refusing to accept?
+- If not, is there a sensible compromise or middle ground that would be appropriate?
+- If not, is there one of the people in the discussion more senior within the context the others (e.g. a track maintainer discussing with a contributor)? If so, that senior person's opinion will be respected.
+- If not, they will make a decision as they feel best - often attempting to find the path that involves the least ongoing changes/disruption.
+
+Our perfect situation would be to never interfer in contributions and let everything come to a natural conclusion.
+However, we have discovered (the hard way) that protracted discussions are often a huge drain on the mental health of our staff and maintainers, and often results in a worse outcome for the student, where someone "gives in", rather than a true consensus being reached.
+If an administrator enters your conversation wearing their "administrator hat", please understand that their intentions are to protect our contributors and students, and move things forward.
+Please treat them with respect, and take any decisions they make as final.
+Do not take them entering a conversation to be a sign of disrepsect - when an administrator acts in that role it does not necessarily mean that they think the conversation has become negative, but they want to protect everyone involved before that happens.
+They will be (attempt to) wield this power pragmatically.
+
+If you feel a conversation is becoming draining and unproductive, or could move that way, please feel free to tag @exercism/maintainers-admin in a post to ask for guidance or reach out to one of the admins on Slack.
+
+Exercism's current administrators are:
+
+- Jeremy Walker (@iHiD on Slack)
```

Comment:
```suggestion
- Jeremy Walker (@iHiD on Slack and GitHub)
```

Reply:
Then again, you might want to not add this change if your intention is that people ONLY use @exercism/maintainers-admin